src/components/ItemPicker/Item.vue

```diff
@@ -13,13 +13,13 @@
       </slot>
     </b>
     <font-awesome-icon
-      v-if="selected && !disabled"
+      v-if="selected && !disabled && !hideIcons"
```

Comment:
wrap both <font-awesome-icon> with <template> and check for hidden & disabled there
also, _hideIcons computed prop that checks if disabled & hidden.